### ISSUE TITLE
Sometimes area pagmark resizing causes an internal error

### ISSUE THREAD

**Issue by krassowski** [burtonator/polar-bookshelf#329]
It appears that this happens only if another area pagemark (the one created with meta + click, I am not sure of the terminology here) is already on the same page:
![resizing_page_mark](https://user-images.githubusercontent.com/5832902/47372625-09319f00-d6e2-11e8-94fc-ec39914a9070.gif)
PS. Keep up good work, the app is great! I would love to help on the coding side as well once I get some free time  (this weekend?).

**Comment by krassowski** [comment]
Possible duplicate of #12 (sorry I did not see that earlier)

**Comment by burtonator** [comment]
Yeah.. I have to agree that this is an annoying bug.. The problem is that some of the math around the overlapping boxes has a bug that I'm working on fixing. The code here is still in JS to I need to port it to TS which should help resolve the issue.
